src/app/components/SideNav/SideNav.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useState } from 'react';
+import { Link } from 'react-router-dom';
+
+import styles from './SideNav.module.css';
+
+export type SideNavProps = {
+  children: string;
```

Comment:
Usually, `children` is not only a `string`.
You set the content of the button, which shows the menu.
Maybe call it `showSideNavLabel: string;`?

If this is always the same, you don't need a property for this.